### DISCUSSION TITLE
Have Mobs Ignore OP'd and Creative Players

### DISCUSSION THREAD

**Discussion by EchocallingEarth** [PurpurMC/Purpur#1078] [__Completed__] [answered]
Add an option that would allow OP'd players to be ignored by mobs (Sort of like Spectator mode). This would prevent mobs from spawning around OP'd players or those in creative mode.  This would allow OP'd players or those in creative to work without Mobs spawning near them, It would further allow OP'd players who may not use beds to sleep to be around normal players without attracting mobs to that player due to not sleeping. 

Sorry, if my description is lacking.
If I can provide more on this idea please reach out to me here or in discord EchocallingEarth#0001

**Comment by RhythmicSys** [comment, marked as answer]
Added in 5b1c2b1cf433ce02d3a81ff90c34df4371815d38
Build #1768
